install/lib/class.FroxlorInstall.php

```diff
@@ -1273,6 +1298,48 @@ private function _guessWebserver()
 		}
 	}
 
+
+	/**
+	 * get/guess linux distribution
+	 */
+	private function _guessDistribution()
+	{
+		// post
+		if (! empty($_POST['distribution'])) {
+			$this->_data['distribution'] = $_POST['distribution'];
+		} else {
+			$os_version = parse_ini_file('/etc/os-release', false);
+			if ($os_version['ID'] == 'debian') {
```

Comment:
open if, no content, elseif check for the same statement...

Reply:
oops